### PR TITLE
Recommend replace as a replacement to cutrewrite.

### PR DESCRIPTION
**Pull request by Zimmi48**
As suggested by @thery to Chris Dams on Coq-Club.

### CODE REVIEW

File: doc/sphinx/proof-engine/tactics.rst [hunk 1]
```diff
@@ -2819,19 +2819,12 @@ simply :g:`t=u` dropping the implicit type of :g:`t` and :g:`u`.
       only in the conclusion of the goal. The clause argument must not contain
       any ``type of`` nor ``value of``.
 
-   .. tacv:: cutrewrite <- (@term = @term’)
+   .. tacv:: cutrewrite {| <- | -> } (@term = @term’)
```
Comment (jfehrle) [inline doc/sphinx/proof-engine/tactics.rst:2822]: The full syntax is `cutrewrite {? {| -> | <- } } @one_term {? in @ident }`, with an apparent semantic restriction that @one_term is in the form `(@term__1 = @term__2)`

**Review by jfehrle — APPROVED** [review]
Perhaps drop in the full syntax for cutrewrite?

**Review by ppedrot — APPROVED** [review]
Might be a good time to remove this tactic altogether. It has been deprecated for a long time...

**Comment by Zimmi48** [comment]
> Might be a good time to remove this tactic altogether. It has been deprecated for a long time...

I can do this in a separate PR for 8.13.

**Comment by jfehrle** [comment]
@coqbot: merge now

**Comment by jfehrle** [comment]
@coqbot: merge now

**Pull request merged**